consumer.ts

```diff
@@ -54,6 +54,14 @@ const sendEachMessage = async ({ message }) => {
   return;
 };
 
+// Instanciate `Consumer` function every 45 seconds to avoid running before producer & Kafka does
+const consumerThrottle = async () => {
```

Comment:
Can we also concider adding this as a function that runs if line 72 or line 74 fails in a try catch?
so basically create a function that connects or subscribes 
and in the catch run this time out to wait and try again, assuming the producer is spooling up